.github/workflows/deploy-unstable-app.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         env:
           VITE_REACT_APP_CONTENTFUL_ENV: ${{ vars.CONTENTFUL_ENV }}
           VITE_REACT_APP_CONTENTFUL_SPACE_ID: ${{ vars.CONTENTFUL_SPACE_ID }}
+          VITE_REACT_APP_FF_UPDATE_CSR: ${{ vars.VITE_REACT_APP_FF_UPDATE_CSR }}
```

Comment:
I wonder if "UPDATE_CSR" is the best name for this feature flag, it feels a bit vague... is this just a feature flag for the decimal version of the complaints rating (in which case we could be a bit more specific) or is it for the new version of the data as a whole, in which case I wonder if FF_NEW_CSR_DATA or something might be a bit more explicit?